stable/prometheus-node-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.15.2"
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 0.3.0
+version: 0.3.1
```

Comment:
this adding new functionality I would say to bump to 0.4.0

Reply:
Ah okay. I'll amend to the last commit